setup.py

```diff
@@ -288,12 +273,14 @@ def write_version(filename: str = os.path.join(*["airflow", "git_version"])):
     'nose-timer',
     'parameterized',
     'paramiko',
+    'pydevd',
```

Comment:
?

Reply:
I've added pydevd as developer dependency so that you do not have to install pydevd in the container after you enter it. But this is more for the possible next step (simplified development environment) where one of the things I describe is how to easily debug from within IDE (such as pycharm). I know you don't like IDE's ;) but several people already asked for this and some guidance on how to debug within container. We found that pydevd + IntelliJ play together very well. 

I will remove it for now but will try to justify it when I go to breeze.

Reply:
Done.